src/context/AppContextManager.tsx

```diff
@@ -607,11 +607,14 @@ export default class AppContextManager extends React.Component<Props> {
         this.getFullContext(),
         collectionId
       );
-      this.contextState.collectionLoading = false;
+      const collectionItems = get(collection, "items", []);
       this.contextState.collection = collection;
-      console.log("GOT COLLECTION", collection);
+      this.contextState.collectionItems = collectionItems;
```

Comment:
Don't we already have `this.context.collection.items`? Why do we need to store them again as `this.context.collectionItems`?